packages/plugins/typescript/typescript/tests/typescript.spec.ts

```diff
@@ -1830,6 +1830,17 @@ describe('TypeScript', () => {
       `);
       validateTs(result);
     });
+
+    it('Should contain `// @ts-nocheck`', async () => {
+      const schema = buildSchema(/* GraphQL */ `
+        "My custom scalar"
+        scalar A
+      `);
+      const result = await plugin(schema, [], { enableTsNoCheck: true }, { outputFile: '' });
+
+      expect(result.prepend).toContain(`// @ts-nocheck`);
+      validateTs(result);
+    });
```

Comment:
Does this also work for the near-operation-file and gql-tag-preset and other stuff that emits multiple files?